Gutenberg.podspec

```diff
@@ -14,19 +16,11 @@ Pod::Spec.new do |s|
   s.source_files = 'react-native-gutenberg-bridge/ios/*.{h,m,swift}'
   s.requires_arc = true
   s.preserve_paths = 'bundle/ios/*'
+  s.swift_version = '4.2'
 
-  s.dependency 'React/Core', react_native_version
-  s.dependency 'React/CxxBridge', react_native_version
-  s.dependency 'React/RCTAnimation', react_native_version
-  s.dependency 'React/RCTImage', react_native_version
-  s.dependency 'React/RCTLinkingIOS', react_native_version
-  s.dependency 'React/RCTNetwork', react_native_version
-  s.dependency 'React/RCTText', react_native_version
-  s.dependency 'React/RCTActionSheet', react_native_version
-  s.dependency 'React/DevSupport', react_native_version
+  s.dependency 'React-Core', react_native_version
+  s.dependency 'React-RCTImage', react_native_version
```

Comment:
Reducing these to only what we need means that the build will be leaner and hopefully faster. Transitive dependencies (such as `yoga`) will still be resolved.